keps/sig-network/2593-multiple-cluster-cidrs/README.md

```diff
@@ -956,6 +1070,10 @@ details). For now, we leave it here.
 
 ###### How does this feature react if the API server and/or etcd is unavailable?
 
+The MultiCIDRRangeAllocator will not be able to fetch node objects and thus the
+node will remain in NotReady state as it will have no PodCIDRs assigned. This is
+in line with the current RangeAllocator behavior.
+
```

Comment:
This controller runs in the controller-manager, if the controller manager doesn't connect to the apiserver the cluster will have greater problems 😄

Reply:
Ack, added a summary to the section